cmd/ore/aws/upload.go

```diff
@@ -236,6 +238,21 @@ func runUpload(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if len(uploadTags) > 0 {
+		hvmTagMap := make(map[string]string)
```

Comment:
Rather than duplicate this code you could build a single tagMap and then only the call to `CreateTags` and the error check would be present in both image blocks.